go/chat/storage/inbox.go

```diff
@@ -490,83 +489,78 @@ func (i *Inbox) applyQuery(ctx context.Context, query *chat1.GetInboxQuery, rcs
 			queryConvIDMap[c.String()] = true
 		}
 	}
+
+	queryMemberStatusMap := map[chat1.ConversationMemberStatus]bool{}
```

Comment:
Can we just make this a static thing so we aren't creating it all the time?

Reply:
but it's built from the query

Reply:
Right but 99% of the time it will be the same values, I was thinking we could swap in a map with the defaults values in that case instead of creating it.

Reply:
sgtm